controllers/discountController.js

```diff
@@ -114,13 +114,15 @@ const getActiveDiscounts = async (req, res) => {
         });
 
         if (!activeDiscounts?.length) {
-            return res.status(400).json({ error: "No active coupon" });
+            return res
+                .status(400)
+                .json({ error: "Active discounts not found" });
         }
```

Comment:
@laubana this should not give you an error when discounts are empty. It just means that there is no record of active discounts yet.